web-registry/src/components/organisms/MediaForm.tsx

```diff
@@ -142,224 +168,94 @@ const MediaForm: React.FC<MediaFormProps> = ({ submit, initialValues }) => {
           'regen:galleryPhotos': getURLListInitialValue(
             initialValues?.['regen:galleryPhotos'],
           ),
-          'regen:landStewardPhoto': getURLInitialValue(
-            initialValues?.['regen:landStewardPhoto'],
-          ),
+          'regen:creditText': initialValues?.['regen:creditText'] || '',
           'regen:videoURL': getURLInitialValue(
             initialValues?.['regen:videoURL'],
           ),
         }}
-        validate={async (values): Promise<MediaValuesErrors> => {
-          const errors: MediaValuesErrors = {};
-          if (graphData?.shaclGraphByUri?.graph) {
-            const projectPageData = { ...getProjectPageBaseData(), ...values };
-            const report = await validate(
-              graphData.shaclGraphByUri.graph,
-              projectPageData,
-              'http://regen.network/ProjectPageMediaGroup',
-            );
-            for (const result of report.results) {
-              const path: string | undefined = result.path?.value;
-              if (path) {
-                const compactedPath = getCompactedPath(path) as
-                  | keyof MediaValuesErrors
-                  | undefined;
-                if (compactedPath === 'regen:previewPhoto') {
-                  errors[compactedPath] = { '@value': requiredMessage };
-                } else {
-                  // for gallery photos, display general error message below "Gallery Photos" section
-                  errors['regen:galleryPhotos'] = 'You must add 4 photos';
-                }
-              } else {
-                // or constraint not satisfied on regen:landStewardPhoto/regen:videoURL
-                errors['regen:landStewardPhoto'] = {
-                  '@value': requiredMessage,
-                };
-              }
-            }
-          }
-          return errors;
-        }}
-        onSubmit={async (values, { setSubmitting, setTouched }) => {
-          setSubmitting(true);
-          try {
-            await submit(values);
-            setSubmitting(false);
-            setTouched({}); // reset to untouched
-            if (isEdit && confirmSave) confirmSave();
-          } catch (e) {
-            setSubmitting(false);
-          }
-        }}
+        validate={handleValidate}
+        onSubmit={handleSubmit}
       >
-        {({ submitForm, isValid, isSubmitting, errors, touched }) => {
+        {({ submitForm, isValid, isSubmitting, errors, values, touched }) => {
+          const shouldRenderPhoto = (i: number): boolean => {
```

Comment:
Note: i've set the max amount of gallery photos to 4, similar to the old setup. This is a bit hacky, but it looks for 
1. the presence of the initial preview photo and then 
2. the presence of a prior gallery photo

and renders the next photo box